### ISSUE TITLE
Migrations example from Heroku to Digital Ocean

### ISSUE THREAD

**Issue by ttiras** [hasura/graphql-engine#4986]
The most frequent use case is start dev on heroku free instance and deploy to digital ocean with docker. Config V1 had great explanation and examples on official hasura channel on youtube. 
With new cli-migrations-v2 the docs has been changed and it is hard to follow and decide. 
It could be really nice to have a whole example of steps for switching from heroku to digital ocean, maybe also with some advanced features like security options. 
And one other thing could be nice is to have an example of dockerizing a node-js app along with a framework of choice(react, vue) and hasura. 
These 2 examples could really help hasura users who are new to docker and related tech.

**Comment by marionschleifer** [comment]
This can be a guide under integration/migration tutorials: https://hasura.io/docs/1.0/graphql/core/guides/integrations/index.html

**Comment by adithyaakrishna** [comment]
Hi @marionschleifer Can I work on this issue?
And if so how can I add the docs to the site and is docker compulsory?

**Comment by marionschleifer** [comment]
@adithyaakrishna feel free. Check out our [contributing guide for docs](https://github.com/hasura/graphql-engine/blob/master/docs/CONTRIBUTING.md). And yes, Hasura always runs in a Docker container.

**Comment by adithyaakrishna** [comment]
Okay, Thanks @marionschleifer 
I'm using Win 10 Home Edition, do you have any suggestions on running docker in it or should I switch to Ubuntu through a VM?

**Comment by marionschleifer** [comment]
@adithyaakrishna it would probably be good to run it through a VM with Ubuntu, as then we can cover more users 🙂
